tinkerbell/rufio/templates/deployment.yaml

```diff
@@ -23,6 +23,14 @@ spec:
         control-plane: controller-manager
         stack: tinkerbell
     spec:
+      {{- if .Values.hostNetwork }}
+      hostNetwork: true
```

Comment:
Curious, by this is needed?

Reply:
it was a confusion on my part. I removed the hostNetwork from rufio and reverted that change